src/device/mod.rs

```diff
@@ -8,16 +8,16 @@ pub mod drop_privileges;
 mod integration_tests;
 pub mod peer;
 
-#[cfg(any(target_os = "macos", target_os = "ios"))]
+#[cfg(any(target_os = "macos", target_os = "freebsd"))]
 #[path = "kqueue.rs"]
 pub mod poll;
 
 #[cfg(target_os = "linux")]
 #[path = "epoll.rs"]
 pub mod poll;
 
-#[cfg(any(target_os = "macos", target_os = "ios"))]
-#[path = "tun_darwin.rs"]
+#[cfg(any(target_os = "macos", target_os = "freebsd"))]
+#[path = "tun_bsd.rs"]
```

Comment:
we may want this file name to indicate that it's both darwin and bsd

Reply:
darwin is considered bsd